docs(/src/kubernetes): Initial K8s section for Zeebe Docs using helm.zeebe.io (#3138)
zeebe-io/zeebe: docs/src/kubernetes/README.md

# Zeebe on Kubernetes

> ![Zeebe on K8s](zeebe-k8s-helm.png)

Reviewer: ```suggestion
![Zeebe on K8s](/kubernetes/zeebe-k8s-helm.png)
```

This section covers the fundamentals of how to run Zeebe in Kubernetes. There are several alternatives on how to deploy applications to a Kubernetes Cluster, but the following sections are using Helm charts to deploy a set of components into your cluster. 

Helm allows you to choose exactly what chart (set of components) do you want to install and how these components needs to be configured. These Helm Charts are continously being improved and released to the Official [Zeebe Helm Chart Repository http://helm.zeebe.io](http://helm.zeebe.io)

You are free to choose your Kubernetes provider, our Helm charts are not cloud provider specific and we encourage [reporting issues here](http://github.com/zeebe-io/zeebe-full-helm/issues) if you find them. 

> You can also join us in [Slack at: http://zeebe-io.slack.com](http://zeebe-io.slack.com)

Reviewer: This URL is only useful if you already joined the slack channel, otherwise you have to use the invite app https://zeebe-slack-invite.herokuapp.com/

This document is divided in the following sections:

> - [Prerequisites](prerequisites.md)

Reviewer: ```suggestion
- [Prerequisites](/kubernetes/prerequisites.html)
```

> - [Getting to know and Installing Zeebe Helm Charts](installing-helm.md)

Reviewer: ```suggestion
- [Getting to know and Installing Zeebe Helm Charts](/kubernetes/installing-helm.html)
```

> - [Accessing Operate from outside our Kubernetes Cluster](accessing-operate.md)

Reviewer: ```suggestion
- [Accessing Operate from outside our Kubernetes Cluster](/kubernetes/accessing-operate.html)
```

# Zeebe on Kubernetes

![Zeebe on K8s](imgs/zeebe-on-k8s.png)

This section covers the fundamentals of how to run Zeebe in Kubernetes. There are several alternatives on how to deploy applications to a Kubernetes Cluster, but the following sections are using HELM charts to deploy a set of components into your cluster. 

HELM allows you to choose exactly what chart (set of components) do you want to install and how these components needs to be configured. This HELM Charts are continously being improved and released to the [Official Zeebe HELM Chart Repository http://helm.zeebe.io](http://helm.zeebe.io)

You are free to choose your Kubernetes provider, our HELM charts are not cloud provider specific and if we encourage [reporting issues here](http://github.com/zeebe-io/zeebe-full-helm/issues). 

> You can also join us in Slack at: 

Reviewer 2: Missing the Slack address?

Author: @jwulf good catch.. thanks..
